dartagnan/src/test/java/com/dat3m/dartagnan/litmus/LitmusPTXv7_5LivenessTest.java

```diff
@@ -0,0 +1,41 @@
+package com.dat3m.dartagnan.litmus;
+
+import com.dat3m.dartagnan.configuration.Arch;
+import com.dat3m.dartagnan.configuration.Property;
+import com.dat3m.dartagnan.utils.Result;
+import com.dat3m.dartagnan.utils.rules.Provider;
+import com.dat3m.dartagnan.utils.rules.Providers;
+import com.dat3m.dartagnan.wmm.Wmm;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.io.IOException;
+import java.util.EnumSet;
+
+@RunWith(Parameterized.class)
+public class LitmusPTXv7_5LivenessTest extends AbstractLitmusTest {
```

Comment:
Please also add a test class for PTX v6.0

Reply:
Added